db.sql

```diff
@@ -0,0 +1,88 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TYPE CloudCover AS ENUM (
+  'Percent50',
+  'Percent70',
+  'Percent80',
+  'Any'
+);
+
+CREATE TYPE ImageQuality AS ENUM (
+  'Percent20',
+  'Percent70',
+  'Percent85',
+  'Any'
+);
+
+CREATE TYPE SkyBackground AS ENUM (
+  'Percent20',
+  'Percent50',
+  'Percent80',
+  'Any'
+);
+
+CREATE TYPE WaterVapor AS ENUM (
+  'Percent20',
+  'Percent50',
+  'Percent80',
+  'Any'
+);
+
+CREATE TYPE TargetObjectType AS ENUM (
+  'Sidereal',
+  'NonSidereal'
+);
+
+CREATE TYPE ObsStatus AS ENUM (
+  'New',
+  'Included',
+  'Proposed',
+  'Approved',
+  'ForReview',
+  'Ready',
+  'Ongoing',
+  'Observed'
+);
+
+CREATE TABLE constraints (
+  id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
```

Comment:
We don't need to change this but there is a requirement for "memorable" urls. we may want to use a human readable id eventually/aditionally

Reply:
Ok. We can change to anything as long as it's unique.